src/modules/45-dashboards/strings/strings.en.yaml

```
@@ -43,6 +43,10 @@ editModal:
   editDashboard: 'Edit Dashboard'
   submitFail: 'Edit dashboard failed, Please try again.'
   success: 'Successfully updated the dashboard'
+cloneDashboardModal:
+  title: 'Clone Dashboard'
+  submitFail: 'Clone dashboard failed, Please try again.'
+  success: 'Successfully cloned the dashboard'
```

Might read better re-arranged
```suggestion
  submitFail: 'Clone dashboard failed, Please try again.'
  success: 'Dashboard successfully cloned'
```